tools/cli/commands/create.py

```diff
@@ -280,6 +280,7 @@ def flags(parser):
         default=_DATALAB_DEFAULT_DISK_SIZE_GB,
         help='size of the persistent disk in GB.')
 
+    """ Broken due to changes to the notebook server, disable until we fix it.
```

Comment:
Let's just remove it for now and then add it back when we are ready with the fix.

It will still be available in the git history, so there's no need to keep it commented out in the code.